src/loader-components/ExampleTemplate.module.scss

```diff
@@ -34,12 +34,13 @@
     justify-content: flex-end;
   }
 
+  .css,
   .code {
     background-color: var(--terra-dev-site-example-template-code-background-color, #f6f8fa);
     height: 100%;
   }
 
-  .bg-toggle,
+  .css-toggle,
```

Comment:
these buttons need focus styles added for keyboard navigation.

Reply:
Added focus logic here: [2e28aa6](https://github.com/cerner/terra-dev-site/pull/281/commits/2e28aa6167c3cf422dbd74674a56591b9ecbdf9b), and the actual styles here: [f086e80](https://github.com/cerner/terra-dev-site/pull/281/commits/f086e8097bc5393e9187cd5ca9efb848e1dcb773).